src/glossary.rst

```diff
@@ -1,18 +1,23 @@
 Glossary
 ========
 
+.. set the default scope for cylc references to suite.rc so we can do [runtime]
+   rather than suite.rc[runtime]
+
+.. cylc-scope:: suite.rc
+
 .. glossary::
    :sorted:
 
    suite
    Cylc suite
-      A Cylc suite is a directory containing a ``suite.rc`` file which contains
+      A Cylc suite is a directory containing a :cylc:conf:`suite.rc` file which contains
       :term:`graphing<graph>` representing a workflow.
 
 
    suite directory
       The suite directory contains all of the configuration for a suite e.g.
-      the ``suite.rc`` file.
+      the :cylc:conf:`suite.rc` file.
```

Comment:
Stuff like this is going to conflict every single time with my #142 PR 😅

Reply:
We'll try to get yours in first then!

Reply:
For now yes. Ronnies change should be merged soon so I'll get the rebase!